core/src/main/java/com/sequenceiq/cloudbreak/service/ServiceEndpointCollector.java

```diff
@@ -193,7 +205,7 @@ public String getManagerServerUrl(Cluster cluster, String managerIp) {
             if (ExposedService.HIVE_SERVER.equals(exposedService) || ExposedService.HIVE_SERVER_INTERACTIVE.equals(exposedService)) {
                 return getHiveJdbcUrl(gateway, managerIp);
             } else if (ExposedService.NAMENODE.equals(exposedService) && versionComparator.compare(() -> stackVersion, () -> "2.6") > 0) {
-                return getHdfsUIUrl(gateway, managerIp, privateIps.get(ExposedService.NAMENODE.getAmbariServiceName()).iterator().next());
+                return getHdfsUIUrl(gateway, managerIp, privateIps.get(ExposedService.NAMENODE.getCmServiceName()).iterator().next());
```

Comment:
Is not this backwards incompatible with Ambari?

Reply:
No, according to what we agreed on with @keyki, most probably we are unable to launch Ambari based clusters on master line, so it is not needed.

Reply:
Okay. But then it might make sense to eventually remove all references to Ambari (which is out of scope of this PR, of course).